gammapy/image/core.py

```diff
@@ -932,3 +1014,17 @@ def __str__(self):
             info += self[name].__str__()
             info += '\n'
         return info
+
+
+def _get_resampled_wcs(wcs, factor, downsampled):
```

Comment:
FYI: The wcs resampling helper function could go here:
https://github.com/gammapy/gammapy/blob/master/gammapy/utils/wcs.py

But I don't care if you do this now, I'll look at
https://github.com/fermiPy/fermipy/blob/master/fermipy/wcs_utils.py
and our utils soon and then start making PRs to move things into the right places / make them simple and consistent / add tests.